src/JuliusSweetland.OptiKey/UI/ViewModels/Management/WordsViewModel.cs

```diff
@@ -147,7 +147,8 @@ public bool UseDefaultKeyboardLayoutIsVisible
                        || KeyboardAndDictionaryLanguage == Enums.Languages.EnglishUK
```

Comment:
Could you please change this to instead check UseAlphabeticalKeyboardLayoutIsVisible || UseSimplifiedKeyboardLayoutIsVisible || EnableCommuniKateKeyboardLayoutIsVisible